pages/api/heroes/index.js

```diff
@@ -0,0 +1,36 @@
+import { heroes } from '../../../data/heroes';
+import { PrismaClient } from '@prisma/client';
+
+const prisma = new PrismaClient();
+
+export default async function handler(req, res) {
+  if (req.method === 'GET') {
+    // GET METHOD
+    const heroesData = await prisma.heroes.findMany();
+    res.status(200).json(heroesData);
+  } else if (req.method === 'POST') {
+    // POST METHOD
+    const heroData = req.body;
+    heroData.map(async (data) => {
+      const Heroes = await prisma.heroes.create({
+        data: data,
+      });
+    });
+    res.status(200).json(heroData);
+  } else if (req.method === 'DELETE') {
+    // DELETE METHOD
+    const heroIds = req.body.id;
+    heroIds.map(async (heroId) => {
+      try {
+        const Heroes = await prisma.heroes.delete({
```

Comment:
```suggestion
        const heroes = await prisma.heroes.delete({
```

untuk nama variable, ckup gini aja yak @berry1611